app/middleware/tracking.js

```diff
@@ -0,0 +1,28 @@
+import AppConstants from 'constants/AppConstants';
+
+export const track = (trackData) => {
```

Comment:
- [ ] [Use function declarations instead of function expressions](https://github.com/airbnb/javascript#functions--declarations).